dockertest/environment.py

```diff
@@ -7,12 +7,17 @@
        in autotest!
 """
 
+import sys
 import errno
 import subprocess
-# This always fails on travis with: no get in xattr (E0611)
-# dispite successful 'pip install pyxattr==0.5.1'.  A test
-# for this name has been added to .travis.yml directly.
-from xattr import get as getxattr  # pylint: disable=E0611
+try:
+    # TODO: Get python-selinux module working in travis (ubuntu)
+    import selinux
+except ImportError:
```

Comment:
IOW: I don't think you need an exception handler. You can just try to import selinux, and not care if it fails, because: